presto-tests/src/test/java/io/prestosql/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -730,7 +730,18 @@ public void testDescribeInput()
         MaterializedResult expected = resultBuilder(session, BIGINT, VARCHAR)
                 .row(0, "unknown")
                 .row(1, "bigint")
-                .row(2, "varchar")
+                .row(2, "varchar(25)")
+                .build();
+        assertEqualsIgnoreOrder(actual, expected);
+
+        session = Session.builder(getSession())
+                .addPreparedStatement("my_query", "SELECT 1 from (VALUES (CHAR 'Pi', TIMESTAMP '2012-03-14 1:59:26.535', DECIMAL '3.14')) AS t (t_varchar, t_timestamp, t_decimal) WHERE t_varchar = ? AND t_timestamp = ? AND t_decimal = ?")
+                .build();
+        actual = computeActual(session, "DESCRIBE INPUT my_query");
+        expected = resultBuilder(session, BIGINT, VARCHAR)
+                .row(0, "char(2)")
+                .row(1, "timestamp(3)")
```

Comment:
can you have a test with  timestamp(6) as well as unbound varchar?

Reply:
I have added test for timestamp and unbounded varchar